### PR TITLE
homework-02-Bojana

### PR DESCRIPTION
**Pull request by BojanaMR**

### CODE REVIEW

File: basic-table.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head></head>
+<body>
+   <table border="2" width="300" height="200">
+       <thead>
+           <tr>
+               <th colspan="3">Section_1</th>
+           </tr>
+       </thead>
```
Comment (dpeshevski) [inline basic-table.html:10]: bez `<thead>`, toj del stavi go vo `<tbody>`

Comment (BojanaMR) [reply basic-table.html:10]: 👍 Vo red ke go popravam .
Ostanato e se vo red ?

File: layout.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                <td>2 p.m.</td>
+                <td>5 p.m.</td>
+            </tr>
+            <tr>
+                <td>Wednesday</td>
+                <td>8 a.m.</td>
+                <td>12 p.m.</td>
+                <td>xsl formatting object</td>
+            </tr>
+        </tbody>
```
Comment (Nikolina5) [inline layout.html:55]: Super e Bojana. Bravo. Mnogu citlivo i super! :) <3

File: layout.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (Nikolina5) [inline layout.html:3]: Zabelezav samo deka ti fali <title> </title> tag

File: table-nested-02.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                <th>Header column 2</th>
+                <th>Header column 3</th>
+                <th>Header column 4</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>Row 2 item 1</td>
+                <td>Row 2 item 2</td>
+                <th>Row 2 nested table 1
```
Comment (Nikolina5) [inline table-nested-02.html:19]: Tuka mislam ne treba <th> tag , ovaa nested tabela e premnogu kratka...

The <th> tag defines a header cell in an HTML table.  :)
www.w3schools.com

**Review by Nikolina5 — COMMENTED** [review]
Bojana super si! Samo napred !! :) 🥇

**Review by Nikolina5 — APPROVED** [review]
Bojana super si. Samo napred !! :) <3 🥇

File: layout.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                <td>Introduction to xml</td>
+            </tr>
+            <tr>
+                <td>Validity: dtd and relax NG</td>
+            </tr>
+            <tr>
+                <td rowspan="5">Tuesday</td>
+                <td rowspan="2">8:00 a.m.</td>
+                <td rowspam="2">11:00 a.m.</td>
+                <td rowspan="3">XPath</td>
```
Comment (Metodija-Mitovski) [inline layout.html:35]: interesno resenie :)

File: layout.html (hunk 2 above)
Comment (BojanaMR) [reply layout.html:55]: Thanks 😊

File: table-nested-02.html (hunk 4 above)
Comment (BojanaMR) [reply table-nested-02.html:19]: Vo pravos si :).
Thanks a lot  ! :)

**Comment by VesnaRosh** [comment]
Super indentacija Bojana, keep it up :)

**Review by emillija — APPROVED** [review]
good job Bojana

**Review by VesnaRosh — APPROVED** [review]

**Review by dpeshevski — APPROVED** [review]

**Pull request merged**